migrations/tenant/0017-note-teams@chage-role-type.sql

```diff
@@ -0,0 +1,2 @@
+-- Change role type to note teams from integer to string
+ALTER TABLE public.note_teams ALTER COLUMN role TYPE varchar(255);
```

Comment:
maybe `role` should be enum? `varchar(255)` looks strange

Reply:
i turned back to integer. I tried to change to enum, but migration brakes trying to cast integer to enum